app/models/admin/import.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# A non-activerecord helper class for csv upload
+class Admin::Import
```

Comment:
If I understand correctly, this will never clean up uploaded CSV files after they're processed since the model is not persisted but nothing triggers paperclip to destroy the data? The thing is, if you're going to process the file synchronously, you don't even need paperclip, you could just read the tempfile from the upload.

Reply:
Yeah that sounds like a better way to do it, actually.

Reply:
@ClearlyClaire Is this something you want to change over at glitch-soc and have me merge into this PR or should I take a look at making the change direectly on mastodon?

Reply:
This is actually more work than I anticipated, but I should be able to do it. The current code is also a bit more wonky than I thought